### ISSUE TITLE
How to increase beams and token limit?

### ISSUE THREAD

**Issue by DarkAlchy** [WASasquatch/was-node-suite-comfyui#262]
, standing in front of a

That normally means it ran out of tokens but I don't see where to increase it nor a way to get more beams so it is more accurate (temprature, top_p, etc.... too).

**Comment by DarkAlchy** [comment]
def generate(self, image, sample=False, num_beams=3, max_length=30, min_length=10, top_p=0.9, repetition_penalty=1.0):
        image_embeds = self.visual_encoder(image)
        
Do we have to adjust that instead of a front interface?

**Comment by WASasquatch** [comment]
Yeah I did not expose that, but I get quite longer captions so I feel
something else is tripping it up there.

**Comment by DarkAlchy** [comment]
No matter what I tried it always stopped right there in mid sentence.

**Comment by WASasquatch** [comment]
> No matter what I tried it always stopped right there in mid sentence.

Even increasing beam size? What is the image? Some images just trip out captioning. Like scifi you often get " sci fi fi fi fi fi fi fi fi fi fi fi fi" into infinity until end of limit.

**Comment by DarkAlchy** [comment]
I have had it do it on some others as well.  The problem is this is Blip instead of blip2 or llama which do this far less often.
